lib/python/ensembl/brc4/runnable/dump_stable_ids.py

```diff
@@ -0,0 +1,420 @@
+#!env python3
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+from os import path
+from typing import Any, List, Dict
+from datetime import datetime
+
+from ensembl.brc4.runnable.core_server import core_server
+
+
+class UnsupportedEvent(ValueError):
+    pass
+
+
+class StableIdEvent:
+    """Represents a stable id event from one gene set version to another one. Various events:
+    - new genes
+    - deleted genes
+    - merged genes (several genes to one)
+    - split genes (one gene to several)
+    - mixed (several genes to several)
+
+    Args:
+    from_list = List of genes the previous gene set
+    to_list = List of genes in the new gene set
+    release = new gene set release name
+    date = date of the new gene set
```

Comment:
```suggestion
    Attributes:
        from_list: List of genes the previous gene set.
        to_list: List of genes in the new gene set.
        release: New gene set release name.
        date: Date of the new gene set.
        name: ...
        pairs: ...
```
Google docstring formatting.